vignettes/get_started.R

```diff
@@ -0,0 +1,59 @@
+## ----setup, include = FALSE----------------------------------------------
```

Comment:
looks like this was auto-generated by knitr and accidentally committed, could you please delete it and recommit?

Reply:
Sorry, this request wasn't clear. Could you please delete (and remove from git tracking) the *.R files that were generated by knitr? They aren't used by the package but clutter up the directory.